OS-SPEC.md

```diff
@@ -0,0 +1,23 @@
+# Linux
+
+Linux has both file paths and syscall as part of its ABI. In order to correctly run most applications additional parts of the environment must be setup.
```

Comment:
The Linux ABI consists of several special file paths and syscalls.
Most applications targeted at Linux will require additional parts of the environment to be set up to run correctly.